jujugui/static/gui/src/app/components/added-services-list-item/test-added-services-list-item.js

```diff
@@ -563,29 +530,17 @@ describe('AddedServicesListItem', function() {
         unfadeService={unfadeService}
         changeState={sinon.stub()}
         getUnitStatusCounts={sinon.stub()}
-        service={service} />);
+        service={mockService} />);
 
-    // Check that the container has the proper classes when no visibility
-    // toggle is active.
-    assert.equal(
```

Comment:
Because the classes don't change anymore within the scope of these tests. All the AddedServicesListItem object does now is call the injected methods (focusService, unfocusService, etc.). These methods are now stubbed out and asserted to make sure they're being called with the right data, but since they never execute, the ListItem component doesn't re-render, doesn't get new classes, etc.

That sort of testing now takes place in `test-added-services-list.js`, in the "…performs the necessary work to…" tests.